.github/workflows/main.yml

```diff
@@ -40,6 +40,9 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
```

Comment:
the runner in this job installs node 18, which I believe has some compatibility issues with openSSL inside this windows job. I downgraded it to node 16 to get the build to pass ([stackoverflow for reference](https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported))

Reply:
Have we tried running on [windows server 2022](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources)? Would also suggest adding a comment here so we know why node 16 was installed when we try to upgrade.